common/setup.py

```diff
@@ -0,0 +1,30 @@
+import os
+from setuptools import find_packages, setup
+
+# allow setup.py to be run from any path
+os.chdir(os.path.normpath(os.path.join(os.path.abspath(__file__), os.pardir)))
+
+setup(
+    name="data-refinery-common",
+    version="0.1.0",
+    packages=find_packages(),
+    install_requires=["django>=1.10.6",
+                      "data_refinery_models>=0.1.0",
+                      "boto3>=1.4.4"],
+    license="BSD License",
+    description="Common functionality to be shared between Data Refinery sub-projects.",
+    url="https://www.greenelab.com",
+    author="Kurt Wheeler",
+    author_email="kurt.wheeler91@gmail.com",
```

Comment:
You may want to use the lab's email address instead. Ask Rene about it. I don't have it at hand now.